connectivity/tests/health.go

```diff
@@ -45,21 +45,14 @@ func runHealthProbe(ctx context.Context, t *check.ConnectivityTest, pod *check.P
 
 		stdout, err := pod.K8sClient.ExecInPod(ctx, pod.Pod.Namespace, pod.Pod.Name, "", cmd)
 		if err != nil {
-			select {
-			case <-done:
+			t.Warnf("cilium-health probe failed: %q, stdout: %q, retrying...", err, stdout)
+		}else{
```

Comment:
nit: looks like you didn't run gofmt, please do.

Reply:
I ran gofmt and squashed the commits